### ISSUE TITLE
Possible coding error in Integration Testing topic 3.x sample app

### ISSUE THREAD

**Issue by guardrex** [dotnet/AspNetCore.Docs#18400]
&gt; Reason I am asking is because I can see `ConfigureWebHost` in the 3.x samples, and in the doc @docs.microsoft.com I have ASP.NET Core 3.1 selected.

https://github.com/dotnet/AspNetCore.Docs/pull/18391#discussion_r427949441

Thanks @michal-ciechan ... I'll investigate that asap, but it probably won't be until early next week.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 77889d43-fedb-f8b4-8fea-379ec946555c
* Version Independent ID: f499b131-8b14-7e10-1728-543fdd5a5656
* Content: [Integration tests in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-3.1)
* Content Source: [aspnetcore/test/integration-tests.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/test/integration-tests.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-test**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by michal-ciechan** [comment]
Just FYI, I did run the 3.x code locally, and it did run as per docs ( `Startup.ConfigureSerivces` first, followed by the Factory `builder.ConfigureServices` ). I guess even htough the method is `ConfigureWebHost` I guess it is still using the `Generic Host` path.

Just my observation as I was curious if it actually works.

**Comment by guardrex** [comment]
@michal-ciechan I checked ... I don't see a virtual `ConfigureHost` method, just `ConfigureWebHost`.

<img width="557" alt="Capture" src="https://user-images.githubusercontent.com/1622880/82673064-da094980-9c06-11ea-9715-c114a04ba122.PNG">

It does seem like the naming "ConfigureHost" would be better. Idk why it kept its naming for 3.x. Perhaps, it would have been a breaking change to do so.

Anyway ... I think we're good after the update that we made. Thanks again!
